### PR TITLE
Feature/events api endpoint

### PR DESCRIPTION
**Pull request by dakriy**
#27 MUST BE MERGED FIRST!

Create an api endpoint for the events. It just returns all events from the database with no filtering or anything like that.

### CODE REVIEW

**Comment by blakestone95** [comment]
I'm going to wait to review until your other stuff is merged... Then I can see what ACTUALLY changed.

**Comment by blakestone95** [comment]
Hey thanks for naming your branches with the "feature/".  This helps us maintain good GitFlow conventions.

File: Legion.BackEnd/tests/Unit/EventTest.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use App\User;
+use App\Event;
+use Carbon\Carbon;
+
+class EventTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /** @test */
+    public function it_creates_an_announcement_by_a_user()
```
Comment (blakestone95) [inline Legion.BackEnd/tests/Unit/EventTest.php]: Function names weren't updated after copypasta :P

Comment (dakriy) [reply Legion.BackEnd/tests/Unit/EventTest.php]: Oh rip. Let me get that.

Comment (blakestone95) [reply Legion.BackEnd/tests/Unit/EventTest.php]: No worries, I know how that goes, believe me xD

Comment (dakriy) [reply Legion.BackEnd/tests/Unit/EventTest.php]: Fixed

**Pull request merged**